graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/DXGraphQLError.java

```diff
@@ -84,23 +82,36 @@ public String getMessage() {
     }
 
     @Override
-    @JsonIgnore
-    public ErrorType getErrorType() {
-        return errorType;
-    }
-
-    @JsonProperty("errorType")
-    public String getRealErrorType() {
+    public ErrorClassification getErrorType() {
         if (errorType != null) {
-            return errorType.toString();
+            return errorType;
```

Comment:
I think we need to wrap errorType here to a SimpleClassification object as well?

Reply:
> I think we need to wrap errorType here to a SimpleClassification object as well? errorType is declared as an ErrorType enum object

ErrorType is already an ErrorClassification